docs/guides/exchange.md

```diff
@@ -151,13 +153,14 @@ Once the backend storage is created, individual accounts for individual users ca
 
 Each account is related to a specific IOTA network (mainnet/devnet) which is referenced by a node property, such as node url (in this example, the Chrysalis devnet balancer).
 
-For more information about `clientOptions`, please refer to [Wallet NodeJs API Reference](https://wiki.iota.org/wallet.rs/libraries/nodejs/api_reference).
+For more information about `clientOptions`, please refer to the [Wallet NodeJs API Reference](https://wiki.iota.org/wallet.rs/libraries/nodejs/api_reference).
 
-The `Alias` can be whatever fits to the given use case and should be unique. The `Alias` is usually used to identify the given account later on. Each account is also represented by an `index` which is incremented (by 1) every time a new account is created. Any account can be then referred to via `index`, `alias`, or one of its generated `addresses`.
+The `Alias` can be whatever fits to the given use case and should be unique. The `Alias` is typically used to identify the given account later on. Each account is also represented by an `index` which is incremented (by 1) every time a new account is created. Any account can then be referred to via `index`, `alias`, or one of its generated `addresses`.
```

Comment:
```suggestion
The `Alias` can be whatever fits the given use case and should be unique as it will be used to identify the account. Each account is also represented by an `index` which is incremented (by 1) every time a new account is created. Any account can then be referred to using its `index`, `alias`, or one of its generated `addresses`.
```